extensions/resteasy-reactive/rest-client-reactive/runtime/src/main/java/io/quarkus/rest/client/reactive/runtime/RestClientRecorder.java

```diff
@@ -9,15 +10,24 @@
 @Recorder
 public class RestClientRecorder {
     private static volatile Map<String, String> configKeys;
+    private static volatile Set<String> blockingClassNames;
```

Comment:
I would say that the name of the field is a little too generic

Reply:
I named it too generic on purpose since we will need it in other use cases (for other client response filters).